btr-web/btr-common-components/composables/useBcrosAuth.ts

```diff
@@ -33,7 +33,7 @@ export const useBcrosAuth = () => {
   }
 
   /** Setup keycloak / user auth pieces */
-  async function setupAuth (kcConfig: KeycloakConfig) {
+  async function setupAuth (kcConfig: KeycloakConfig, currentAccountId?: string) {
```

Comment:
made this change to avoid the console warning about using useRoute in the middleware